docs/UserGuide.md

```diff
@@ -42,13 +47,14 @@ It tracks and leverages on key metrics and information to create an optimal inte
 
 #### What our app manages
 
-InternHunter allows the management of three data types:
+InternHunter allows the management of four data types:
 
-Data type | Alias | Attributes
-----------|-------|-----------
-Company | com | company name, industry, job title
-Internship | int | company name, job title, industry, requirements, period, wage, status
-Profile | me | category, descriptors
+Data type | Alias | Attributes | What it represents
+----------|-------|------------|-------------------
+Company | com | company name, industry, job title | A company offering internships
+Internship | int | company name, job title, industry, requirements, period, wage, status | An internship offered by a company
```

Comment:
I believe these attributes for Company and Internship are the outdated ones?

Reply:
Thanks for the spot, will update it.